backend/core/src/listings/listings.service.ts

```diff
@@ -93,6 +93,14 @@ export class ListingsService {
     }
   }
 
+  public async listIncludeExternal(params: ListingsQueryParams): Promise<Pagination<Listing>> {
```

Comment:
Can this whole function just be one line?

`return await this.list(params)`?

Reply:
Yes it can be. My thought was that it would be further built out in the next iteration, but fine making this a one liner. Updated.